tests/test_outputs/test_intram_by_pg.py

```diff
@@ -1,178 +1,210 @@
-            [6, 1, "E", 41.0],
-            [7, 1, "AA", 0.0],
-            [8, 1, "AD", 143.0],
-            [9, 1, "J", 138.0],
-        ]
-        input_df = pd.DataFrame(data=data, columns=columns)
-        return input_df
-
-    @pytest.fixture(scope="session")
-    def pg_detailed_df(self) -> pd.DataFrame:
+        ni_columns = ["reference", "instance", "201", "211"]
+        ni_data = [[1, 1, "C", 27.0],
+                   [1, 2, "G", 102.0],
+                   [2, 1, "AA", 250.0],
+                   [3, 1, "I", 628.0],
+                   [4, 1, "D", 18.0],
+                   [5, 1, "AA", 7.0],
+                   [6, 1, "E", 41.0],
+                   [7, 1, "AA", 0.0],
+                   [8, 1, "AD", 143.0],
+                   [9, 1, "J", 138.0]]
+        ni_input_data_df = pd.DataFrame(data=ni_data,
+                                        columns=ni_columns)
+
+        return ni_input_data_df
+
+    def pg_detailed_mapper_data(self) -> pd.DataFrame:
         """pg_detailed mapper, including a subset of PGs"""
-        columns = [
+        pg_mapper_columns = [
             "ranking",
             "pg_alpha",
             "Detailed product groups (Alphabetical product groups A-AH)",
-            "Notes",
-        ]
-        data = [
+            "Notes"]
+        pg_mapper_data = [
             [1, "total", "Total", "Total q211 across all PG"],
-            [4, "C", "Food products and beverages; Tobacco products", "Total q211 for PG C"],
-            [5, "D", "Textiles, clothing and leather products", "Total q211 for PG D"],
-            [6, "E", "Pulp, paper and paper products; Printing; Wood and straw products", "Total q211 for PG E"],
+            [4, "C", "Food products and beverages; Tobacco products",
+             "Total q211 for PG C"],
+            [5, "D", "Textiles, clothing and leather products",
+             "Total q211 for PG D"],
+            [6, "E",
+             "Pulp, paper and paper products; Printing; Wood and straw products",
+             "Total q211 for PG E"],
             [8, "G", "Chemicals and chemical products", "Total q211 for PG G"],
             [10, "I", "Rubber and plastic products", "Total q211 for PG I"],
             [11, "J", "Other non-metallic mineral products", "Total q211 for PG J"],
             [27, "Z", "Construction", "Total q211 for PG Z"],
             [28, "AA", "Wholesale and retail trade", "Total q211 for PG AA"],
-            [31, "AD", "Miscellaneous business activities; Technical testing and analysis", "Total q211 for PG AD"],
-            [35, "AH", "Software Development", "Total q211 for PG AH"]
-        ]
-        mapper = pd.DataFrame(data=data, columns=columns)
-        return mapper
+            [31, "AD",
+             "Miscellaneous business activities; Technical testing and analysis",
+             "Total q211 for PG AD"],
+            [35, "AH", "Software Development", "Total q211 for PG AH"]]
+        pg_detailed_mapper = pd.DataFrame(data=pg_mapper_data,
```

Comment:
we're not restricted to 88 chars for tests, so we normally use the full line width rather than breaking lines for dataframes.

Reply:
I've addressed this for all the dataframes.